src/main/resources/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>topvis</title>
+</head>
+<body>
+    <div id="root"></div>
```

Comment:
Could you please specify margin for the contents? Now text sticks to the edge of the screen, which makes it harder to read.